packages/compass-home/src/index.d.ts

```diff
@@ -0,0 +1,6 @@
+declare module 'mongodb-ns' {
```

Comment:
Yeah probably worth updating. I think I'd like to not do it here if that's cool. We might want to have a discussion on bringing that package into the monorepo, and probably doing some updating of it since I think it's not handling special characters like dots in collections names properly, and some of the validation looks not quite up to date. We also might just want to simplify it to just being an interface for db name and collection name if we're not using the other aspects of it.
https://docs.mongodb.com/manual/reference/limits/#naming-restrictions
https://github.com/mongodb-js/ns/blob/master/index.js
I can create a ticket, but I think we might want to discuss where/how we'd like it first. Let's also chat on that tomorrow?